src/NUnitFramework/mock-assembly/MockAssembly.cs

```diff
@@ -113,12 +113,12 @@ public static void Main(string[] args)
         [Category("FixtureCategory")]
         public class MockTestFixture
         {
-            public const int Tests = 10;
-            public const int Suites = 1;
+            public const int Tests = 13;
+            public const int Suites = 2;
```

Comment:
Adds three tests with the test cases below and one additional suite because the test cases form a suite.